src/sessions/VirtualPageLoadTimer.ts

```diff
@@ -240,13 +240,11 @@ export class VirtualPageLoadTimer extends MonkeyPatched<
     };
 
     private recordRouteChangeNavigationEvent(page: Page) {
-        const virtualPageNavigationEvent: NavigationEvent = {
-            version: '1.0.0',
-            initiatorType: 'route_change',
-            navigationType: 'navigate',
-            startTime: page.start,
+        const virtualPageNavigationEvent = {
+            initiatorType: 'navigation',
+            type: 'navigate',
```

Comment:
Open question: PageManager overwrites this field with the non-W3C field 'route_change'. I recommend we avoid overwriting and copy PerformanceAPI exactly, but up for discussion

Reply:
We currently don't use this field in the navigation event for any queries so there aren't any breaking changes to the system. But I would image there was a use case for this when we were implementing support for SPAs. Probably a good idea to get Reno to review this change. 

On the other hand, assuming customer experience breaking changes are okay for v2.x version bump, we could start with strictly W3C compliant fields and come back to add/overwrite this later down the road if necessary.